src/auth/session/storage/custom.ts

```diff
@@ -1,19 +1,20 @@
 import {Session} from '../session';
+import {SessionInterface} from '../types';
 import {SessionStorage} from '../session_storage';
 import * as ShopifyErrors from '../../../error';
 
 export class CustomSessionStorage implements SessionStorage {
   constructor(
-    readonly storeCallback: (session: Session) => Promise<boolean>,
-    readonly loadCallback: (id: string) => Promise<Session | Record<string, unknown> | undefined>,
+    readonly storeCallback: (session: SessionInterface) => Promise<boolean>,
+    readonly loadCallback: (id: string) => Promise<SessionInterface | Record<string, unknown> | undefined>,
```

Comment:
Is `Record<string, unknown` a valid return type for the `loadCallback` method?  Wouldn't the two valid return types be either `SessionInterface` or `undefined`?

Reply:
[This](https://github.com/Shopify/shopify-node-api/pull/126) PR explains why this option was added.